src/libos/src/process/do_spawn/mod.rs

```diff
@@ -223,10 +277,21 @@ fn new_process(
         let elf_name = elf_path.rsplit('/').collect::<Vec<&str>>()[0];
         let thread_name = ThreadName::new(elf_name);
 
-        ProcessBuilder::new()
+        let mut parent;
+        let mut process_builder = ProcessBuilder::new();
+        if reuse_tid.is_some() {
+            // must be in execve syscall
+            // assert!(reuse_vm.is_some());
```

Comment:
Enable the assertion or remove it

Reply:
This assertion is not correct anymore. Removed.